Vizzuality/heco-invest
# Let 553 fe project developer can edit their project developer account

This PR adds the project-developer/edit page

## Testing instructions

Acceptance criteria

The project developer must be signed in

A project developer can see their account information

My area

There must be a way for Project Developers to edit their account information

The functionality to edit Project Developers accounts will be at the Project developer’s “My area”

These are the fields that can be edited: See [reqs doc for specifications](https://docs.google.com/spreadsheets/d/1qtP-qkm8iS6OlluiJwRvQoi6rCns9iKhRYoNzc9qkGg/edit#gid=1884416048)

There will be a way to validate the information entered in the fields

Errors are displayed if the information is incorrect

There will be a way confirm or dismiss the edition of the account

Note: Save/cancel

The layout must comply with the designs defined at: [HeCo - Work in progress](https://www.figma.com/proto/KC49uMR9t1YARmFR0Sr6aN/HeCo---Work-in-progress?node-id=1326%3A25176&viewport=396%2C48%2C0.06&scaling=scale-down&starting-point-node-id=1326%3A25176&show-proto-sidebar=1) 

Note: language is not editable

The changes made by PDs don’t need to be approved to be visible to other users

## Tracking

[LET-553](https://vizzuality.atlassian.net/browse/LET-553)
